src/pages/signup-affiliates/index.tsx

```diff
@@ -86,7 +90,13 @@ const AffiliateSignup = () => {
 
     useEffect(() => {
         trackEvent({ action: 'open' })
-        return () => trackEvent({ action: 'close' })
+        const handleBeforeUnload = (event) => {
+            event.preventDefault()
+            event.returnValue = 'Are you sure you want to leave this page?'
```

Comment:
Shall this be translated?

Reply:
This is banner which occur when user want to close tab
It's my mistake that I asked chatGPT how to handle it and didn't check that value
![Screenshot 2023-12-26 at 15 02 05](https://github.com/binary-com/deriv-com/assets/103182473/d1957526-d898-4ea0-ad91-aa82af3d0d58)